CMakeLists.txt

```diff
@@ -29,9 +29,11 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 set(CMAKE_MACOSX_RPATH ON)
 enable_testing()
-set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+if (NOT MATERIALX_BUILD_JS)
```

Comment:
Hi @JGamache-autodesk , Can you explain this. I'm not sure why this was added and why the JS build should not do this ?

Reply:
On Windows, this makes sure all the executables and DLLs when building shared libraries are put in the same folder. This allows running unit tests without having to add a large number of entries to the PATH environment variable.

Without this, the DLLs end up in every source folder that creates a shared library and materialXTest.exe ends up not being able to find them.

Not an issue on Linux/Mac since the full path to the libraries are stored at link time (and stripped at install time).

Now, the JS build seems to have steps with hardcoded library paths. This is why we do not gather libraries to a common location for this build only.